### ISSUE TITLE
DDOS - does lusca contains protection from ddos attacks

### ISSUE THREAD

**Issue by maxprog** [krakenjs/lusca#115]
Does lusca contains protection from ddos attacks?

**Comment by grawk** [comment]
No, it does not. I would recommend some type of rate limiting middleware in the express application. Else, attempt to detect the DDOS closer to the "edge" of your deployment. I.e. if you are reverse proxying node.js with nginx, at that layer instead.

**Comment by maxprog** [comment]
Thank You very much. I have the same opinion.  I use express-rate-limit and  described on http://maxprog.net.pl/node-js/node-js-protection-from-brute-force-and-ddos-attacks/
